back/engines/commercial/id_nemlog_in/spec/requests/nemlog_in_verification_spec.rb

```diff
@@ -119,4 +119,37 @@
 
     expect(response).to redirect_to('/whatever-page?verification_error=true&error=no_token_passed')
   end
+
+  context 'when verification method is used by Copenhagen' do
+    before do
+      configuration = AppConfiguration.instance
+      settings = configuration.settings
+      settings['verification']['verification_methods'][0]['issuer'] = 'https://kobenhavntaler.kk.dk'
+      configuration.save!
+    end
+
+    it 'does not verify an under 15-year-old' do
+      saml_auth_response.extra.raw_info.attributes['https://data.gov.dk/model/core/eid/age'][0] = '14'
+
+      get "/auth/nemlog_in?token=#{token}&random-passthrough-param=somevalue&pathname=/some-page"
+      follow_redirect!
+
+      expect(response).to redirect_to('/authentication-error?sso_pathname=%2Fsome-page&sso_response=true&error_code=not_entitled_under_15_years_of_age')
```

Comment:
Because the FE has no implementation for verification errors, but does have for SSO errors.
See [Slack thread](https://citizenlabco.slack.com/archives/CQAGMHKT3/p1694605712950699).

Reply:
Reverted to using verification specific redirect URL format(s)